lib/binaries.sh

```diff
@@ -87,10 +66,21 @@ install_yarn() {
 }
 
 install_nodejs() {
-  local version=${1:-16.x}
+  local version="${1:-}"
   local dir="${2:?}"
   local code resolve_result
 
+  if [[ -z "$version" ]]; then
+    # Node.js 18+ is incompatible with ubuntu:18 (and thus scalingo-18) because of a libc mismatch:
+    # node: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.28' not found (required by node)
+    # Fallback to a 16.x default for scalingo-18 until scalingo-18 or Node.js 16.x are EOL.
+    if [[ "$STACK" == "scalingo-18" ]]; then
```

Comment:
You didn't took into account exactly my message ;)



```suggestion
    if [[ "$STACK" == "heroku-18" || "$STACK" == "scalingo-18" ]]; then
```